netbench/cdk/src/main/java/com/aws/PeeringStackProps.java

```diff
@@ -0,0 +1,111 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.aws;
+
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+import software.amazon.awscdk.Environment;
+import software.amazon.awscdk.services.ec2.Vpc;
+
+public interface PeeringStackProps extends StackProps {
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    Vpc getVpcClient();
+
+    Vpc getVpcServer();
+
+    String getStackType();
+
+    String getRef();
+
+    String getCidr();
+
+    String getRegion();
+
+    public static class Builder {
```

Comment:
Did you consider using something like https://projectlombok.org/ to reduce this boilerplate code?

Reply:
I didn't thanks for the link! Do you think this could be an optimization that I look into after completing the basic functionality of my project? Since I don't have too much time during the internship, making sure that the CDK app works well on time is my first priority.

Reply:
Yep, that makes sense